robocop/checkers/naming.py

```diff
@@ -662,3 +669,62 @@ def find_similar_variables(self, tokens, node):
                     col=token.col_offset,
                 )
             self.variables[normalized_token].add(token.value)
+
+
+class DeprecatedStatementChecker(VisitorChecker):
+    """Checker for deprecated statements."""
+
+    reports = (
+        "deprecated-statement",
+    )
+    # deprecated:alternative
+    deprecated_statements_rf3 = {}
+    # FixMe(Malik): Before releasing, swap contents of rf4 to rf5, rf3 = rf4 = {}
```

Comment:
That's because currently the tests will not work if the 5.0 is not released - it's fine to leave it (I will replace it right before release) or it can be fixed now and I will merge with failed build

Reply:
I swapped the values now, tests for rf5 are currently not running, but once we use the 5-version the tests would ran